docs/feature_unicode.md

```diff
@@ -13,16 +13,16 @@ Supports Unicode up to 0xFFFFFFFF. You need to maintain a separate mapping table
 And you may want to have an enum to make reference easier.  So you'd want to add something like this to your keymap:
 
 ```c
-enum unicode_name {
-  BANG, // ‽
-  IRONY, // ⸮
-  SNEK // snke 🐍
+enum unicode_names {
+  BANG,
+  IRONY,
+  SNEK,
 };
 
 const uint32_t PROGMEM unicode_map[] = {
-  [BANG]      = 0x0203D, // ‽
-  [IRONY]     = 0x02E2E, // ⸮
-  [SNEK]      = 0x1F40D // snke 🐍
+  [BANG]    = 0x203D,  // ‽
+  [IRONY]   = 0x2E2E,  // ⸮
+  [SNEK]    = 0x1F40D, // 🐍
```

Comment:
Why the comma here?

Reply:
Trailing commas are a recommendation (and often a rule) in most good style guides out there. The two biggest benefits of trailing commas are: it makes it easier to move lines around; and it reduces diffs and possible merge conflicts when appending items to a list.

That being said, these are docs, and it isn't necessarily required here, but the docs should always contain good code (“lead by example”).